packages/react-components/src/components/LocalVideoTile.tsx

```diff
@@ -249,7 +252,9 @@ export const _LocalVideoTile = React.memo(
           displayName={displayName}
           initialsName={initialsName}
           styles={videoTileStyles}
-          onRenderPlaceholder={props.participantsCount === 1 ? onRenderAvatarOneParticipant : onRenderAvatar}
+          onRenderPlaceholder={
```

Comment:
Is there a reason this lives in the video tile? For the lobby/connecting overlay messages its just over the gallery, I'd assume there would be logic we would be reusing for both?

Reply:
The UI is different when camera in on/ off. When camera is on it's a white text and when camera is off, it's an overlay and need to remove the avatar

Reply:
We need to know about camera status yes, but as a UI component - why do we show a message that's very specific to the composite experience? We shouldn't assume everytime a localvideotile is showing and alone in a call that that translates to "waiting for others to join a call".

Can this be moved to the composite layer? And reuse the lobby logic? This keeps styling etc. consistent. The screens are identical except for the text message:

| | lobby | call (alone) |
| -- | -- | -- |
| no camera | ![image](https://github.com/user-attachments/assets/3650681f-8c17-49ea-85e4-b3224db8b719) | ![image](https://github.com/user-attachments/assets/c8621238-e1d8-4d36-957e-11793127cb96) |
| camera on | ![image](https://github.com/user-attachments/assets/8ec263ab-2913-4e62-bba5-c254246b3543) | ![image](https://github.com/user-attachments/assets/20ad9593-70ae-4f13-9ee4-bd31e50eeebb) |

Reply:
Hmm just looking at the code that could be tricky, the lobby isn't using the videogallery but instead just a single tile...

Reply:
Can we still move the logic to the composite? Unless you feel this should be the default experience for everyone using the video gallery?

Reply:
This is what Ezra requested so I think component and composite should be consistent? It's overall an UI improvement to show the waiting text when there is only one person in the call

Reply:
Chatted offline, this is already in stable so we can't move. If there's a reason someone wishes this to not be shown we can add a new boolean flag to disable